src/components/Shared/ModalPopup.scss

```diff
@@ -0,0 +1,56 @@
+.modalWindow {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%,-50%);
+  transition: all .25s ease;
+  border-radius: 15px 15px 15px 15px;
+  z-index: 99999999999999999;
+}
+
+.header {
```

Comment:
Can we use nesting (as a general best practice, clarity, robustness)?  I think all the classes here might go under `.ModalWindow`.

Reply:
is nesting better? I often find it more confusing by far

Reply:
My take:  It is less confusing for me.  The nesting of classes maps 1-to-1 with the nesting of html elements using the classes, so it is easier to find the classes associated with specific elements, and vice-versa.

You know exactly there a class is supposed to be used, and nesting enforces this in the compiler.

It cleans up situations where there are duplicate class names used in different contexts.  

Nesting keeps the scss nice and clean, less verbose -- in the editor you can collapse a whole hierarchy of classes that don't relate to the ones you're using.

Reply:
so everything should always be nested fully within every single other top level component? it ends up becoming very hard to figure out what is nested where at what level. i guess it would be clean if you perfectly mapped the DOM nesting to the SCSS nesting, but that would be a huge task to get there in our current codebase. and also it feels kind of pointless to nest everything inside the wrapper component, it just adds unnecessary levels of indentation.

Reply:
I don't understand "fully within every single other top level component".  At any rate, feel free to do this as you feel best.  :-)